content-resources/src/main/resources/flyway/oskari/V2_11_0__add_style_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS oskari_maplayer_style (
+    id bigserial NOT NULL,
+    layer character varying(64) NOT NULL,
```

Comment:
I guess this can't really be configured as a foreign key if we want to use -1 for default styles in the same table.

Reply:
We are still considering if we want to use the same table for styles on myplaces/userlayers as well. Leaning towards "no" at the moment so this could be int but it's still up for discussion.

Reply:
This is the first step towards getting end-users styles saved to the database instead of them being just runtime styles.

Reply:
my bad didnt read the todo list